modules/site/src/app/examples/pagination/pagination-basic-usage.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  beforeEach,
+  beforeEachProviders,
+  describe,
+  expect,
+  it,
+  inject,
+} from '@angular/core/testing';
+import { ComponentFixture, TestComponentBuilder } from '@angular/compiler/testing';
+import { Component } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { PaginationBasicUsageComponent } from './pagination-basic-usage.component';
+
+describe('Component: PaginationBasicUsage', () => {
+  let builder: TestComponentBuilder;
+
+  beforeEachProviders(() => [PaginationBasicUsageComponent]);
+  beforeEach(inject([TestComponentBuilder], function (tcb: TestComponentBuilder) {
+    builder = tcb;
+  }));
+
+  it('should inject the component', inject([PaginationBasicUsageComponent],
```

Comment:
I don't think I realized we were unit testing these examples. As it is... these tests (and I'm looking at the others, too, not just this one) seem like they are testing Angular 2 (creating and injecting the component), not really the component behavior itself. What's the point of these?